charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -39,7 +39,7 @@ data:
     # CORS header will ONLY be set if the Origin header was sent. The browser
     # does NOT set the Origin header on GETs: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin
     # Due to this change Access-Control-Allow-Origin '*' is never set.
-    Header set Access-Control-Allow-Origin %{ORIGIN_SUB_DOMAIN}e env=ORIGIN_SUB_DOMAIN
+    Header always set Access-Control-Allow-Origin %{ORIGIN_SUB_DOMAIN}e env=ORIGIN_SUB_DOMAIN
```

Comment:
If it works, it works. But after reading the Apache docs on `always` vs `onsuccess` I'm still not sure it's correct. For example, it sounds like always is not a superset of onsuccess. http://httpd.apache.org/docs/current/mod/mod_headers.html

Reply:
In that case, reality doesn't seem to match the doc.  I suspect there's maybe more nuance since the endpoint that wasn't returning the headers was a success (204) but WAS an OPTIONS request, which seems heuristically to not set header for those requests unless it's set to always